### PR TITLE
[qa] Fix pyton syntax in rpc tests

### PR DESCRIPTION
**Pull request by maflcko**
This is fixing four python syntax errors in the rpc tests. The rest is just removing imports or semicolons.

### CODE REVIEW

**Comment by laanwj** [comment]
This is quite invasive. Wouldn't syntax errors cause the tests to not run at all?

File: qa/rpc-tests/disablewallet.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 from test_framework.test_framework import BitcoinTestFramework
 from test_framework.util import *
 
+
```
Comment (GIJensen) [inline qa/rpc-tests/disablewallet.py]: nit: This additional newline makes this a double-newline. This doesn't seem to match the style of the other files.

**Comment by GIJensen** [comment]
utACK 77779948, this looks like good cleanup to me.

**Comment by MarcoFalke** [comment]
@laanwj Indeed this is touching 129 lines, but all of it should be uncontroversial:

* Python will silently ignore trailing semicolons and I don't feel strong about this change. Still, I think we should not promote such usage.
* Excessive imports won't hurt performance too much but they fuzz the code, imo.
* Finally, python will only complain about syntax errors on runtime. Luckily all of the syntax errors behave like they are not there: You can do `raise AssertionFailure("Failed to mine a version=4 blocks")` to raise a `NameError: name 'AssertionFailure' is not defined` which makes travis fail as well but I think `raise AssertionError("Failed to mine a version=4 blocks")` is much clearer.

**Comment by luke-jr** [comment]
Concept ACK

**Comment by laanwj** [comment]
@MarcoFalke Thanks for the explanation, makes sense.
utACK

**Comment by fanquake** [comment]
Ran this through the extended test suite, no issues.

File: qa/rpc-tests/disablewallet.py (hunk 1 above)
Comment (MarcoFalke) [inline qa/rpc-tests/disablewallet.py]: Not sure where this came from. Still, PEP suggests two empty lines, so I will just leave it as is.

**Pull request merged**